manage.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import os
+import sys
+
+if __name__ == '__main__':
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'listings_app.settings')
```

Comment:
We will be using this file a lot! It helps us start up the app locally, create admin users, make migrations and much more! Read about the Django admin utility here: https://docs.djangoproject.com/en/2.1/ref/django-admin/